tests/tests_pytorch/callbacks/test_lr_monitor.py

```diff
@@ -626,3 +626,39 @@ def configure_optimizers(self):
     assert list(lr_monitor.last_momentum_values) == ["lr-Adam/pg1-momentum", "lr-Adam/pg2-momentum"]
     assert all(val == momentum for val in lr_monitor.last_momentum_values.values())
     assert all(all(val == lr for val in lr_monitor.lrs[lr_key]) for lr_key in lr_monitor.lrs)
+
+
+def test_lr_monitor_early_stopping(tmpdir):
+    """Test that the `LearningRateMonitor` logs are accessible by the `EarlyStopping` callback."""
+
+    class TestModel(BoringModel):
+        def configure_optimizers(self):
+            optimizer = torch.optim.SGD(self.layer.parameters(), lr=0.1)
+            lr_scheduler = torch.optim.lr_scheduler.StepLR(optimizer, step_size=1, gamma=0.5)
+            return [optimizer], [lr_scheduler]
+
+    monitor_key = "lr-SGD"
+    stop_threshold = 0.02
+    expected_stop_epoch = 3
+
+    lr_monitor = LearningRateMonitor()
+    lr_es = EarlyStopping(
+        monitor=monitor_key, mode="min", stopping_threshold=stop_threshold, check_on_train_epoch_end=True
+    )
+    trainer = Trainer(
+        default_root_dir=tmpdir,
+        callbacks=[lr_monitor, lr_es],
+        max_epochs=5,
+        limit_val_batches=0,
+        limit_train_batches=2,
+        logger=CSVLogger(tmpdir),
```

Comment:
Let's set
```suggestion
        logger=False,
```
explicitly because the feature doesn't require a logger to monitor for ES, right?

Reply:
The LearningRateMonitor callback requires it by default.

Reply:
Ah ok. Well in this case we should keep it but consider making it a warning in the future (not this PR) because the monitor callback now in theory is useful without a logger.